regolith/builders/cpbuilder.py

```diff
@@ -43,8 +43,8 @@ def construct_global_ctx(self):
         rc = self.rc
         gtx['people'] = sorted(all_docs_from_collection(rc.client, 'people'),
                                key=position_key, reverse=True)
-        gtx['grants'] = sorted(all_docs_from_collection(rc.client, 'grants'))
-        gtx['groups'] = sorted(all_docs_from_collection(rc.client, 'groups'))
+        gtx['grants'] = all_docs_from_collection(rc.client, 'grants')
+        gtx['groups'] = all_docs_from_collection(rc.client, 'groups')
```

Comment:
Why is this no longer sorted?

Reply:
I don't know if they needed to be sorted in the first place, and the sorting was causing issues with the tests.

Reply:
TypeError: '<' not supported between instances of 'ChainMap' and 'ChainMap'

I guess this needs a sort key